devicetypes/smartthings/aeon-siren.src/aeon-siren.groovy

```diff
@@ -97,9 +97,23 @@ def updated() {
 def parse(String description) {
 	log.debug "parse($description)"
 	def result = null
-	def cmd = zwave.parse(description, [0x98: 1, 0x20: 1, 0x70: 1])
-	if (cmd) {
-		result = zwaveEvent(cmd)
+	if (description.startsWith("Err")) {
```

Comment:
This reminds me, we should probably add something at the hub firmware to allow for the user to attempt secure inclusion without having to exclude and include to attempt again. I will make a note of it for future work, and we can discuss how/steps to bring this for users.

Reply:
The spec for S0 requires exclude on failed secure inclusion, but we should look into it for S2

Reply:
Oh yeah, good point. It be nice to show an icon next to devices that failed secure inclusion to remind the user to do that.